public/app/features/alerting/unified/components/bridges/DeclareIncidentButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { createBridgeURL, PluginBridge, BridgeSupportedPlugin } from '@grafana/runtime';
+import { Button, Tooltip } from '@grafana/ui';
+
+interface Props {
+  title?: string;
+  severity?: 'minor' | 'major' | 'critical';
+}
+
+export const DeclareIncident: FC<Props> = ({ title = '', severity = '' }) => {
+  const history = useHistory();
+  const bridgeURL = createBridgeURL(BridgeSupportedPlugin.Incident, '/incidents/declare', { title, severity });
+
+  return (
+    <PluginBridge
+      plugin={BridgeSupportedPlugin.Incident}
+      loadingComponent={
+        <Button size="sm" type="button" disabled>
+          Declare Incident
+        </Button>
+      }
+      notInstalledComponent={
```

Comment:
Does it mean we want to display `disabled` button all the time? Is it supposed to be a kind of incentive for users to install Incident?

Reply:
Yep, that is correct – it's up to the consumer of the component to decide that.